cusub_perception/cusub_perception_bringup/launch/perception_sub.launch

```diff
@@ -1,35 +1,45 @@
 <launch>
   <arg name="sub" default="leviathan" />
-  <arg name="show_image" default="false" />
 
   <group ns="$(arg sub)">
-    <arg name="launch_prefix" default=""/>
-    <arg name="detect_sleep_time" default="0.0"/>
-    <!-- Config and weights folder. -->
-    <arg name="yolo_weights_path"          default="$(find darknet_cv)/launch/yolo_network_config/weights"/>
-    <arg name="yolo_config_path"           default="$(find darknet_cv)/launch/yolo_network_config/cfg"/>
-    <!-- Load parameters -->
-    <rosparam command="load" ns="darknet_ros" file="$(find darknet_cv)/launch/yolo_network_config/ros.yaml"/>
-    <rosparam command="load" ns="darknet_ros" file="$(find darknet_cv)/launch/yolo_network_config/yolov3-tiny.yaml"/>
-
     <!-- Configurations -->
     <rosparam command="load" file="$(find localizer)/config/localizer_config.yaml"/>
     <rosparam command="load" file="$(find darknet_multiplexer)/config/darknet_multiplexer.yaml"/>
+    <!-- dynamic yolo params -->
```

Comment:
I don't like the duplication of default args but I had to include these at the top level also so that they can be passed command line

Reply:
It seems like you cant do this:
RLException: Invalid <arg> tag: arg 'show_image' has already been declared. 

Arg xml is <arg default="false" name="show_image"/>
The traceback for the exception was written to the log file

Reply:
I'm pretty sure they can be passed over the command line without being top level.

Reply:
fixed show_image bug in sim, confirmed the duplicates are in fact necessary